libs-scala/struct-json/struct-circe-json/src/main/scala/com/daml/struct/circe/StructEncoderDecoder.scala

```diff
@@ -0,0 +1,53 @@
+// Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.struct.circe
```

Comment:
As discussed off-band, circe library will be moved to the downstream project that needs it

Reply:
as the lib is also needed in canton, we've decided to keep it